pkg/volume/fc/fc_util.go

```diff
@@ -101,15 +140,28 @@ func scsiHostRescan(io ioHandler) {
 	}
 }
 
+// check scsi disk is really exist and is attached to the node using SCSI Inquiry
+func checkDiskExist(b fcDiskMounter, disk, wwid string) error {
+	out, err := b.plugin.execCommand("/lib/udev/scsi_id", []string{"--page=0x83", "--whitelisted", "--device=", disk})
```

Comment:
Are there a file we should define dependent commands? or Should't we use external command?